src/components/Accordion/PAccordionTrigger.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <AccordionHeader class="flex">
+    <AccordionTrigger
+      v-bind="delegatedProps"
+      class="p-accordion-trigger"
+      :class="props.class"
+    >
+      <slot />
+      <slot name="icon">
+        <p-icon
+          icon="ChevronDownIcon"
+          class="p-accordion-trigger-icon"
+        />
+      </slot>
+    </AccordionTrigger>
+  </AccordionHeader>
+</template>
+
+<script setup lang="ts">
+  import {
+    AccordionHeader,
+    AccordionTrigger,
+    type AccordionTriggerProps
+  } from 'radix-vue'
+  import { type HTMLAttributes, computed } from 'vue'
+
+  const props = defineProps<AccordionTriggerProps & { class?: HTMLAttributes['class'] }>()
+
+  const delegatedProps = computed(() => {
+    // eslint-disable-next-line id-length, no-unused-vars
+    const { class: _, ...delegated } = props
+
+    return delegated
+  })
```

Comment:
You should use the [`useAttrsStylesAndClasses` ](https://github.com/PrefectHQ/prefect-design/blob/main/src/compositions/attributes.ts#L14)composition for this. 

That'll give you the classes, styles, and attrs as separate reactive refs so you can bind classes and styles to the root and attrs to the child.

Reply:
You might want to set `inheritAttrs: false` too if you don't want any attrs to fallthrough to the root element.